content/posts/web-components.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Portable Spectrograms and Verification Interface"
+date: 2025-03-27T22:00:00+10:00
+image: "/images/ecosounds-banner-image.jpg"
```

Comment:
we used this image on the last blog post? let's find something different

Reply:
I've decided to use the following image that I found in our OneDrive

![ecoacoustic-verification](https://github.com/user-attachments/assets/81cf5c57-58a9-4f38-a479-1d89b194b668)

Reply:
Does it have attribution information?

Reply:
The information in the `attribution.txt` says that "The photo is of unknown origin".

Alternatively, if using this image has an attribution problem, we previously supplied this image to the ABC

https://www.abc.net.au/news/2019-11-26/sound-observatory-google-map-of-animal-sounds/1171842

![image](https://github.com/user-attachments/assets/6c43cfe4-d55f-408e-922a-a21cd397a665)

I could probably track down where this image came from and supply attribution for this image

Reply:
We should track down attribution for whatever we use